build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs

```diff
@@ -219,35 +221,9 @@ private void DeleteBlob(string path)
             _blobContainer.GetBlockBlobReference(path).DeleteAsync().Wait();
         }
 
-        public static string CalculateFullUrlForFile(string file, Product product, string version)
-        {
-            return $"{s_dotnetBlobRootUrl}/{CalculateRelativePathForFile(file, product, version)}";
-        }
-
         private static string CalculateRelativePathForFile(string file, Product product, string version)
         {
             return $"{product}/{version}/{Path.GetFileName(file)}";
         }
-
-        public static async Task DownloadFile(string blobFilePath, string localDownloadPath)
```

Comment:
Yeah.  The methods `CalculateFullUrlForFile` and `DownloadFile` do not appear to be used anywhere.  It made sense to remove them in this PR because they are using a hard-coded "dotnet" container and would have to be refactored if we were to keep them.